src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -61,7 +61,16 @@ int CondTraverseToString(const OpBase *ctx, char *buff, uint buff_len) {
 	offset += snprintf(buff + offset, buff_len - offset, "%s | ", op->op.name);
 	offset += QGNode_ToString(op->ae->src_node, buff + offset, buff_len - offset);
 	if(op->ae->edge) {
-		if(op->ae->operands[0].transpose) {
+		// Determine whether thise edge should be printed left-to-right or right-to-left.
+		bool transpose = false;
```

Comment:
can't we used `op->transposed_edge` set in the `Init` function?

Reply:
No, the EXPLAIN pathway doesn't call op `Init` routines.

Reply:
👍